### ISSUE TITLE
Use provided version to manage semantic versioning checks

### ISSUE THREAD

**Issue by vrozov** [siom79/japicmp#108]
Instead of explicitly specifying what may change between version, it should be possible to extract this information from jar version. If the last two digits in the version are zeros, it is major version change and API may change, if only the last digit is zero, it is minor version change and binary compatibility should be preserved, if the last digit is not zero, it is a patch and API may not change at all.

**Comment by siom79** [comment]
I have added a new option `breakBuildBasedOnSemanticVersioning` to the maven plugin. This option tries to deduce the change level (major, minor, patch, unchanged) from the version (assuming it uses three digits) and breaks the build if this does not match the changes detected.

```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout develop
mvn install
```

**Comment by siom79** [comment]
Released with version 0.7.0.
